src/segment-loader.js

```diff
@@ -155,6 +155,14 @@ export default class SegmentLoader extends videojs.EventTarget {
     }
   }
 
+  static get GOAL_BUFFER_LENGTH() {
+    return GOAL_BUFFER_LENGTH;
+  }
+
+  static set GOAL_BUFFER_LENGTH(v) {
+    GOAL_BUFFER_LENGTH = v;
+  }
```

Comment:
There is really little reason to make this a getter/setter combo. We tend to avoid them except where required for compatibility reasons.